cmake/onnxruntime_common.cmake

```diff
@@ -37,7 +37,7 @@ source_group(TREE ${REPO_ROOT} FILES ${onnxruntime_common_src})
 
 add_library(onnxruntime_common ${onnxruntime_common_src})
 
-onnxruntime_add_include_to_target(onnxruntime_common gsl date)
+onnxruntime_add_include_to_target(onnxruntime_common gsl tz)
```

Comment:
Do we need the 'tz' include to get the date headers?

Reply:
You've probably tried this, but could we just add the 'date_interface' library instead given this is what it has in the cmake file:

  add_library(date_interface INTERFACE) # an interface (not a library), to enable automatic 
include_directory (for when just date.h, but not "tz.h and its lib" are needed)